docs/products/flink/concepts/flink-features.rst

```diff
@@ -0,0 +1,40 @@
+Aiven for Apache Flink® features
+================================
+Aiven for Apache Flink is powered by the open-source framework Apache Flink, a distributed processing engine for stateful computations over data streams. It enables you to easily get started with real-time stream processing using SQL.
+
+Some of the key features of Aiven for Apache Flink® include:
+
+Flink SQL
+---------
+Apache Flink allows you to develop streaming applications using standard SQL. Aiven for Apache Flink is a fully managed service that provides various features for developing and running streaming applications using Flink on the Aiven platform. 
+One of these features is the SQL editor, which is a built-in feature of the Aiven Console. The built-in SQL editor allows you to easily create and test Flink SQL queries, explore the table schema of your data streams and tables, and deploy queries to your streaming application. This makes it easy to develop and maintain streaming applications using Flink SQL on the Aiven platform.
+
+Build applications to process data
+----------------------------------
+Applications in the Aiven for Apache Flink service are containers that hold everything related to a Flink job, including the source and sink connectors and the data processing logic. They serve as the entry point for executing a Flink job within the Aiven Flink service. The Aiven console provides an intuitive interface and workflow to build applications. You can select the source and sink connectors and create statements for processing the data within your application. Additionally, you can create multiple versions of an application. Applications in Aiven for Apache Flink significantly improve the developer experience and make it easier to build and deploy Flink applications.
+
+Interactive queries
+-------------------
+The interactive query feature in Aiven for Apache Flink allows you to preview the data of a Flink table or job without outputting the rows to a sink table such as Kafka. This can be useful for testing and debugging purposes, as it allows you to quickly and easily see the data that is being processed by your Flink application.
```

Comment:
```suggestion
The interactive query feature in Aiven for Apache Flink allows you to preview the data of a Flink table or job without outputting the rows to a sink table such as Apache Kafka®. This can be useful for testing and debugging purposes, as it allows you to quickly and easily see the data that is being processed by your Flink application.
```

Reply:
Fixed